### PR TITLE
add more subdomains for Kalooga

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

File: src/chrome/content/rules/Kalooga.com.xml [hunk 1]
```diff
@@ -1,10 +1,49 @@
 <!--
-	(www.): works; mismatched, CN: *.squarespace.com
+	cert mismatch:
+		- (www.)
```
Comment (J0WI) [inline src/chrome/content/rules/Kalooga.com.xml]: `www` WFM, ^ can be redirected to `www`.

File: src/chrome/content/rules/Kalooga.com.xml [hunk 2]
```diff
@@ -1,18 +1,62 @@
...
+	<target host="reporter.kalooga.com" />
+	<target host="signup.kalooga.com" />
+	<target host="static.kalooga.com" />
+
 	<target host="geo.kaloo.ga" />
 	<target host="img.kaloo.ga" />
 	<target host="publishing.kaloo.ga" />
 
+	<test url="http://kalooga.com/" />
+	<test url="http://www.kalooga.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Kalooga.com.xml]: Please add a target for this host

File: src/chrome/content/rules/Kalooga.com.xml [hunk 3]
```diff
@@ -1,18 +1,62 @@
...
 	<target host="publishing.kalooga.com" />
+	<target host="reporter.kalooga.com" />
+	<target host="signup.kalooga.com" />
+	<target host="static.kalooga.com" />
+
 	<target host="geo.kaloo.ga" />
 	<target host="img.kaloo.ga" />
 	<target host="publishing.kaloo.ga" />
 
+	<test url="http://kalooga.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Kalooga.com.xml]: Please keep a comment for the issue on this host.

File: src/chrome/content/rules/Kalooga.com.xml [hunk 4]
```diff
@@ -1,18 +1,65 @@
...
+	<test url="http://www.kalooga.com/" />
+
 	<test url="http://publishing.kaloo.ga/wid/16513.js?loc=http%3A%2F%2Fwww.bz-berlin.de%2Fberlin%2Fber-zweite-anzeige-gegen-jochen-grossmann" />
 	<test url="http://img.kaloo.ga/thumb?url=http%3A%2F%2Fwww.shz.de%2Fimg%2Fwirtschaft%2Fcrop11177506%2F8806138589-cv16_9-h335%2Furn-newsml-dpa-com-20090101-151111-99-05409-large-4-3.jpg&amp;md5val=911f772734b61479440330f1140d13b4&amp;key=00f8a403a132c924d4f37b5df7818f1454f668f8&amp;size=192x140" />
 	<test url="http://publishing.kaloo.ga/acct/4455.js" />
 	<test url="http://geo.kaloo.ga/json/" />
+	
+	<rule from="^http://kalooga\.com/"
```
Comment (J0WI) [inline src/chrome/content/rules/Kalooga.com.xml]: Please keep a comment for the issue on this host.

Comment (numismatika) [reply src/chrome/content/rules/Kalooga.com.xml]: cert mismatch is noted now

File: src/chrome/content/rules/Kalooga.com.xml (hunk 2 above)
Comment (J0WI) [reply src/chrome/content/rules/Kalooga.com.xml]: This test urls are now obsolete since you added them as targets.

**Pull request merged**